### ISSUE TITLE
Hyperparameters for training Resnet18 on CIFAR10?

### ISSUE THREAD

**Issue by kshitijsachan** [ain-soph/trojanzoo#187]
Thanks for making this benchmark! You have provided great documentation.

In the trojanzoo paper, you have a table showing that you trained resnet18 on CIFAR10 with 95.37% accuracy? Do you have the hyper parameters for that training run? (e.g. did you use `resnet18`,` resnet18_ap_comp`, `resnet18_comp`, or `resnet18_s`, etc.) I've tried a few different configurations and have been unable to train a model with >90% accuracy.

I've also had trouble reproducing your results in Table 10 of the paper. I am generally getting better performance for STRIP and much worse performance on NEO than you reported, but I would guess this is because I am using different hyper parameters/models. Perhaps you still have the pretrained attacked model weights that you could share? Thanks.

**Comment by ain-soph** [comment]
python ./examples/train.py --color --verbose 1 --dataset cifar10 --model resnet18_comp --lr_scheduler --cutout --save
